examples/PasswordExpiryWatcher/README.md

```diff
@@ -0,0 +1,96 @@
+	- `go install .`
+
+This will install a binary called `passwordExpiryWatcher` to your `$GOBIN`
+
+> If you'd like to be able to call this binary from an arbitrary directory make sure your `$GOBIN` is in your `$PATH` (linux) or `%PATH` (windows)
+
+### To Run
+
+To run this tool you will need to use what is called the "command line". On Windows this is a program called "Power Shell", on macOS it is an app called "Terminal". 
+
+> While running this tool requires no previous experience with either of those programs some might feel wary or nervous working with a tool they don't understand. The following instructions in this section should provide you with all you need to get up and running, but if you would like to learn about the how and why of the command line we highly recommend the excellent (and free!) [Command Line Crash Course by Zed Shaw](http://cli.learncodethehardway.org/). Zed even provides a direct email hotline for users that get stuck. However if you are stuck and would rather just get your users' password expiration dates ASAP please don't hesitate to contact JumpCloud support for assistance running this tool.
+
+This tool only takes two (required) arguments:
+- `key` is your JumpCloud API key
+- `output` is where you'd like to put the resulting CSV file
+
+For example:
+
+macOS/Linux:
+
+`./PasswordExpiryWatcher -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
+
+Windows:
+
+`./PasswordExpiryWatcher.exe -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
+
+
+> If you have renamed your binary simply replace `PasswordExpiryWatcher` with the new name
+
+> If you installed the binary with the Go toolchain and your `$GOBIN` is in your `$PATH` or `%PATH`, or if you moved the binary to `/usr/local/bin` you can run the above command at any time on your command line excluding the "./"
+
+##### Windows Instructions
+1. Open the program `Power Shell`
+2. Using the `cd` (stands for "Change Directory") command navigate to where you downloaded your binaries
+	- For example, if we downloaded and unzipped the binaries in our `Download` folder we just have to run: `cd Downloads\JumpCloudAPI_Examples_windows_386`. If we downloaded to our desktop the command will probably look something like: `cd Desktop\JumpCloudAPI_Examples_windows_386`
+	- If you used the Go install instructions and your `$GOBIN` is in your `%PATH` you can skip step 2 and go right to 3
+3. Grab your API key from the JumpCloud Admin console
+	- Click on your email on the top right hand corner to access the API Settings
+4. Run the command
+	- `./PasswordExpiryWatcher.exe -key=YOUR_API_KEY_GOES_HERE -output=CSV_FILE_OUTPUT_GOES_HERE`
+	- If you would like your CSV file to go somewhere else besides the current directory make sure you include the _full path_ of the file
+		- Good Example: `-output=C:\Users\MyUser\CSVFiles\password_expirations.csv` 
+		- Bad Example `-output=..\CSVFiles\password_expirations.csv`
```

Comment:
The fact that this is an issue probably indicates that we're doing something wrong in our code. When tools have this issue it drives me nuts (looking at you, OSX).